docs/src/integrate/nodejs-api.md

```diff
@@ -397,9 +397,9 @@ The `LintMessage` value is the information of each linting error. The `messages`
   `true` if this is a fatal error unrelated to a rule, like a parsing error.
 * `message` (`string`)<br>
   The error message.
-* `line` (`number | undefined`)<br>
+* `line` (`number`)<br>
   The 1-based line number of the begin point of this message.
-* `column` (`number | undefined`)<br>
+* `column` (`number`)<br>
```

Comment:
> `line` and `column` can still be `undefined`.

Thanks for the counterexample. Having a type checker hooked up would catch that sort of thing automatically! I've force pushed a new commit sequence that removes aebd80540c505439ce7a39aec0c783ceb8c7ec8a and updates the final commit from c51989400b342623a6a3f365d0ca283405dda8eb to 55bed0864588429248e68d99fa9c5d27c3c66dde so that `line` and `column` are possibly `undefined`.

Reply:
I'd be in favor of leaving `nodeType` off. This is really a vestigial property that doesn't serve any purpose any longer.

Reply:
> I'd be in favor of leaving `nodeType` off. This is really a vestigial property that doesn't serve any purpose any longer.

Agreed.